keycloak/config/quarkus.properties

```diff
@@ -10,4 +10,13 @@ quarkus.log.category."com.github.thomasdarimont.keycloak".level=DEBUG
 # Use quarkus access logging
 # See https://quarkus.io/guides/http-reference#quarkus-vertx-http-config-group-access-log-config_quarkus.http.access-log.enabled
 #quarkus.http.access-log.enabled=true
-#quarkus.http.access-log.pattern=%h %l %u %t "%r" %s %b %m "%{i,Referer}" "%{i,User-Agent}" "%{i,X-Request-Id}" "%{i,X-Organization-Id}" %D
\ No newline at end of file
+#quarkus.http.access-log.pattern=%h %l %u %t "%r" %s %b %m "%{i,Referer}" "%{i,User-Agent}" "%{i,X-Request-Id}" "%{i,X-Organization-Id}" %D
+
+quarkus.opentelemetry.enabled=true
+quarkus.opentelemetry.tracer.enabled=true
```

Comment:
we could also add:
[quarkus.opentelemetry.tracer.resource-attributes](https://quarkus.io/guides/opentelemetry#quarkus-opentelemetry_quarkus.opentelemetry.tracer.resource-attributes)
e.g.:
```
quarkus.opentelemetry.tracer.resource-attributes=service.name=acme-keycloak
```

See: https://quarkus.io/guides/opentelemetry